### ISSUE TITLE
[1.10.2] Recipes requiring XP not showing in JEI when SlabCraft mod is used

### ISSUE THREAD

**Issue by b0bst3r** [rwtema/extrautilities_old_issues#1124]
extrautils2-1.10.2-1.1.1
jei_1.10.2-3.13.4.384

Not much else to say other than Magical wood, Golden Lasso etc not showing recipes.

**Comment by ghost** [comment]
I couldn't reproduce this, although I have seen it before. It may be an interaction with a 3rd mod, possibly related to the XP cost. I suggest you upgrade to the most recent versions if you can and post a mod list if the behavior persists.

**Comment by b0bst3r** [comment]
So you're telling me that you can see recipes for the above items using extrautils2-1.10.2-1.1.1
jei_1.10.2-3.13.4.384+ and yeah on latest JEI as per the norm?

**Comment by ghost** [comment]
Yes. But, I could see them with your versions installed as well so it may be a mod interaction problem. Try the update and if it doesn't work, post the modlist.

**Comment by b0bst3r** [comment]
Yeah it's a clash with slabcraft - http://pastebin.ubuntu.com/23649051/ removing slabcraft shows the recipes.
